big-o-notation/big-o.js

```diff
@@ -0,0 +1,49 @@
+/* exported uniqueQuadratic, uniqueLinear */
+
+function uniqueLinear(words) {
+  const seen = {}; // 1 * 1 = O(1)
+  const unique = []; // 1 * 1 = O(1)
+
+  for (
+    let i = 0; // 1 * 1 = O(1) NESTED LOOP!
```

Comment:
We wouldn't usually describe this as a nested loop. Usually that term is used to refer to one loop being nested inside another loop.